evap/staff/tests/test_json_importer.py

```diff
@@ -503,7 +521,13 @@ def test_import_courses_special_cases(self):
         self.assertEqual(evaluation_life.weight, 1)
 
     def test_import_ignore_non_responsible_users(self):
-        with override_settings(NON_RESPONSIBLE_USERS=["4@example.com"]):
+        with override_settings(
+            NON_RESPONSIBLE_USERS=[
+                "4@example.com",
+                # to test that this filters cleaned emails
```

Comment:
This is approaching this from the wrong angle though, right? It is never meaningful to have unclean emails in the setting. What we should test however is that if a user comes in with an unclean email, but their cleaned email is in the setting, then they still get filtered.